saleor/graphql/order/mutations/utils.py

```diff
@@ -1,24 +1,19 @@
-from typing import List
-
 from django.utils import timezone
 
 from ....order import ORDER_EDITABLE_STATUS
 from ....order.models import Order
 
 
-def invalidate_order_prices(order: Order, *, save: bool) -> List[str]:
+def invalidate_order_prices(order: Order, *, save: bool) -> None:
```

Comment:
Could this function be potentially useful outside of mutations? I'm wondering if we shouldn't move it to `saleor/order/utils.py`.